.env.example

```diff
@@ -0,0 +1,24 @@
+###################################################################
+# App parameters
```

Comment:
I would make it clear that this is (currently?) only used when using the makefile